src/app/SharpRaven/Data/SentryEvent.cs

```diff
@@ -36,29 +36,34 @@ namespace SharpRaven.Data
     /// <summary>
     /// Represents an event being captured by <see cref="IRavenClient.Capture(SentryEvent)"/>.
     /// </summary>
+    [Serializable]
```

Comment:
I was originally serialising sentryevent, so you are correct it doesn't need to be.